tutorials/supervisor/monitor_supervisor.sh

```diff
@@ -0,0 +1,9 @@
+# Start supervisord if is not already running.
```

Comment:
It's reasonable to leave this here for now, but in that case the comment should be deleted or updated to correctly describe what it does. This is copy/pasted from `start_supervisor.sh`.